packages/connect-popup/src/static/popup.html

```diff
@@ -670,16 +670,12 @@ <h3></h3>
             <div class="select-account p2wpkh">
                 <div>
                     <div class="tabs">
-                        <div data-tab="p2wpkh" class="tab-selection p2wpkh">Accounts</div>
+                        <div data-tab="p2wpkh" class="tab-selection p2wpkh">Default accounts</div>
```

Comment:
maybe something like 
```
Segwit accounts (default)
```
?

or whatever, lets wait for @Hannsek

Reply:
"Default accounts (SegWit)" seems fine. 👍🏻

Don't know from the top of my head how we handle SegWit/Segwit in Suite. Please use whatever is in Suite.

Reply:
I'm using the same label as in Suite which doesn't say SegWit, but maybe it's better to mention it

Reply:
Here is a different environment for the users, so let's mention it.

Reply:
However I was talking about whether we use SegWit (big "W") or Segwit (small "w").

Reply:
I think big W is standard